src/main/java/com/greenfox/tribes1/ApplicationUser/DTO/ApplicationUserDTO.java

```diff
@@ -0,0 +1,21 @@
+package com.greenfox.tribes1.ApplicationUser.DTO;
+
+import lombok.*;
```

Comment:
Try to not import all the elements of the libary, just the necessary ones

Reply:
can not resolve this issue now as the current version of the program is not running on my pc .... but will do so!

Reply:
corrected as per requested